proto/decentraland/social/friendships/friendships.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+package decentraland.social.friendships;
+
+// Validation and anti-spam errors
+enum FriendshipErrorCode {
+  TOO_MANY_REQUESTS_SENT = 0;
+  NOT_ENOUGH_TIME_PASSED = 1;
+  BLOCKED_USER = 2;
+  NON_EXISTING_USER = 3;
+  INVALID_REQUEST = 4;
+  UNKNOWN = 5;
+}
+
+enum EventType {
+  REQUEST = 0;
+  ACCEPT = 1;
+  REJECT = 2;
+  CANCEL = 3;
+  DELETE = 4;
+}
+
+// Just a generic empty type
+message Empty {}
+
+// This message is a response that is sent from the server to the client
+message FriendshipEvent {
+  oneof type {
```

Comment:
Might be mistaken here, but maybe `payload` or `body` would be better as it won't be just a discriminator but rather the whole event.